Samples/iOS-SwiftUI/iOS-SwiftUI-UITests/SentryRedactModifierTests.swift

```diff
@@ -0,0 +1,23 @@
+#if canImport(UIKit) && canImport(SwiftUI)
+@testable import Sentry
+@testable import SentrySwiftUI
+import SwiftUI
+import XCTest
+
+class SentryRedactModifierTests: XCTestCase {
```

Comment:
What is the relationship between redaction and TTID/TTFD?

Reply:
This test was moved from `SentryTest` to `iOS-SwiftUI-UITests` because with all the changes to SentrySwiftUI, we cant add it as a SentryTest dependency. It complains about duplicate declarations.